src/db/cass/app.py

```diff
@@ -3,9 +3,14 @@
 from src.file_services import FileServices
 from src.rtmp_saver import RtmpSaver
 from flask import Flask
+from getpass import getpass
 import settings
 
 app = Flask(__name__)
+auth_options = {"loginUrl": settings.LOGIN_URL,
+    "rtmpRequestUrl": settings.RTMP_REQUEST_URL}
+auth_options["username"] = input("Username: ")
+auth_options["password"] = getpass()
```

Comment:
What do I do if docker enters it via environment variable? (It causes error in Docker if you make things interactive.

Reply:
Added environment var check in settings